docs/self-hosted-appcircle/install-server/helm-chart/configuration/docker-k8s-migration.md

```diff
@@ -0,0 +1,822 @@
+---
+title: Docker to Kubernetes Migration
+description: Learn how to migrate from standalone Appcircle server to Kubernetes Appcircle server.
+tags: [self-hosted, helm, configuration, kubernetes, migration]
+sidebar_position: 95
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import SpacetechExampleInfo from '@site/docs/self-hosted-appcircle/install-server/linux-package/configure-server/\_spacetech-example-info.mdx';
+
+## Overview
+
+This guide provides a step-by-step walkthrough for migrating your self-hosted **Appcircle** instance from a standalone Docker environment to a Kubernetes cluster. It assumes that you already have:
```

Comment:
- [ ] Add general caution about user should change any configuration while migration (for instance, SMTP etc.) it's unsupported

If user needs change: first it should migrate successfully then make configuration change using the relevant configuration documentation page.